### PR TITLE
Administration settings to handle expiration rules

### PR DESCRIPTION
**Pull request by vitormattos**

### CODE REVIEW

File: src/views/Settings/ExpirationRules.vue
Comment (giovannism20) [inline src/views/Settings/ExpirationRules.vue:1]: Can u add a print of this modification ?

Comment (vitormattos) [reply src/views/Settings/ExpirationRules.vue:1]: [screen-recorder-wed-dec-27-2023-12-23-47.webm](https://github.com/LibreSign/libresign/assets/1079143/074adaf9-a2d5-4107-bfba-f5af33022294)

Comment (giovannism20) [reply src/views/Settings/ExpirationRules.vue:1]: Is really necessary to have a request on each number update ? I know is not too much updates is something that we  can think about.

**Review by giovannism20 — APPROVED** [review]

File: src/views/Settings/ExpirationRules.vue
Comment (vitormattos) [reply src/views/Settings/ExpirationRules.vue:1]: Isn't necessary but my limited knowledge about JS made me to do by this way. I also disliked this approach to send a request to each key up and down. Maybe a delay would be good but with my limited knowledge I think that the code would be very bad.

Comment (vitormattos) [reply src/views/Settings/ExpirationRules.vue:1]: Maybe we can create a follow up issue to implement an improvement and reduce this, but is a JS from admin side and used only by the sysadmin. I think that isn't a big problem.

Comment (giovannism20) [reply src/views/Settings/ExpirationRules.vue:1]: Yes, we can implement it after. And maybe not a big problem.

Comment (vitormattos) [reply src/views/Settings/ExpirationRules.vue:1]: We need an issue about this. Could you help creating the issue?

Comment (giovannism20) [reply src/views/Settings/ExpirationRules.vue:1]: Issue #2030

**Pull request merged**